lib/trento/settings/api_key_settings.ex

```diff
@@ -8,6 +8,8 @@ defmodule Trento.Settings.ApiKeySettings do
 
   import Ecto.Changeset
 
+  defdelegate authorize(action, user, params), to: Trento.Settings.Policy
```

Comment:
Remember to add this `defdelegate` to the `trento/activity_logging/settings.ex` file as well